kubernetes-fizzbuzz-example/massdriver.yaml

```diff
@@ -35,7 +35,7 @@ params:
         replicaCount: 1
       ingress:
         enabled: true
-        path: /
+        path: /fizzbuzz
```

Comment:
this change is important so users can add another app without configuring dns and use path based routing in nginx